<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="1.212" tests="5" failures="5">
  <testsuite name="Root Suite" timestamp="2025-05-03T15:09:40" tests="0" file="cypress/e2e/features/assignment_editing.feature" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Assignment Editing and Updating" timestamp="2025-05-03T15:09:40" tests="5" time="1.178" failures="5">
    <testcase name="Assignment Editing and Updating Editing an unpublished assignment" time="0.000" classname="Editing an unpublished assignment">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  &gt; 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  > 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130296:76)
    at visitFailedByErr (http://localhost:8000/__cypress/runner/cypress_runner.js:129704:12)
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130279:13)
    at tryCatcher (http://localhost:8000/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:8000/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:8000/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:8000/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:8000/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:8000/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:8000/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10707:12)
    at Registry.runStepDefininition (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9245:48)
    at Object.fn (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10146:41)
    at runStepWithLogGroup (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9850:36)
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10143:60)]]></failure>
    </testcase>
    <testcase name="Assignment Editing and Updating Making limited changes to a published assignment" time="0.000" classname="Making limited changes to a published assignment">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  &gt; 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  > 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130296:76)
    at visitFailedByErr (http://localhost:8000/__cypress/runner/cypress_runner.js:129704:12)
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130279:13)
    at tryCatcher (http://localhost:8000/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:8000/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:8000/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:8000/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:8000/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:8000/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:8000/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10707:12)
    at Registry.runStepDefininition (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9245:48)
    at Object.fn (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10146:41)
    at runStepWithLogGroup (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9850:36)
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10143:60)]]></failure>
    </testcase>
    <testcase name="Assignment Editing and Updating Attempting to reduce deadline of an assignment with submissions" time="0.000" classname="Attempting to reduce deadline of an assignment with submissions">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  &gt; 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  > 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130296:76)
    at visitFailedByErr (http://localhost:8000/__cypress/runner/cypress_runner.js:129704:12)
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130279:13)
    at tryCatcher (http://localhost:8000/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:8000/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:8000/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:8000/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:8000/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:8000/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:8000/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10707:12)
    at Registry.runStepDefininition (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9245:48)
    at Object.fn (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10146:41)
    at runStepWithLogGroup (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9850:36)
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10143:60)]]></failure>
    </testcase>
    <testcase name="Assignment Editing and Updating Adding new file attachments to a published assignment" time="0.000" classname="Adding new file attachments to a published assignment">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  &gt; 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  > 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130296:76)
    at visitFailedByErr (http://localhost:8000/__cypress/runner/cypress_runner.js:129704:12)
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130279:13)
    at tryCatcher (http://localhost:8000/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:8000/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:8000/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:8000/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:8000/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:8000/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:8000/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10707:12)
    at Registry.runStepDefininition (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9245:48)
    at Object.fn (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10146:41)
    at runStepWithLogGroup (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9850:36)
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10143:60)]]></failure>
    </testcase>
    <testcase name="Assignment Editing and Updating Removing grading criteria from a published assignment" time="0.000" classname="Removing grading criteria from a published assignment">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  &gt; 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:8000/login

The response we received from your web server was:

  > 404: Not Found

This was considered a failure because the status code was not `2xx`.

If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130296:76)
    at visitFailedByErr (http://localhost:8000/__cypress/runner/cypress_runner.js:129704:12)
    at <unknown> (http://localhost:8000/__cypress/runner/cypress_runner.js:130279:13)
    at tryCatcher (http://localhost:8000/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:8000/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:8000/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:8000/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:8000/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:8000/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:8000/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:8000/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10707:12)
    at Registry.runStepDefininition (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9245:48)
    at Object.fn (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10146:41)
    at runStepWithLogGroup (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:9850:36)
    at Context.eval (http://localhost:8000/__cypress/tests?p=cypress/e2e/features/assignment_editing.feature:10143:60)]]></failure>
    </testcase>
  </testsuite>
</testsuites>